src/pybind/mgr/dashboard/plugins/debug.py

```diff
@@ -80,10 +81,11 @@ def handler(self, action: Actions):
 
     def custom_error_response(self, status, message, traceback, version):
         self.response.headers['Content-Type'] = 'application/json'
-        error_response = dict(status=status, detail=message, request_id=str(self.request.unique_id))
+        error_response = dict(status=status, detail=html.unescape(message),
```

Comment:
this change deserves a separate commit.

Reply:
You're rigth! This was mostly exploratory work.